js/sign.js

```diff
@@ -0,0 +1,52 @@
+const errorMessageEmail = document.querySelector('.enter-email');
+const errorMessageEmail2 = document.querySelector('.wrong-email');
+const emailInput = document.querySelector('.email-input');
+
+emailInput.onblur = function (e) {
+  const regex = new RegExp('[a-z0-9]+@[a-z]+\.[a-z]{2,3}');
+  if (emailInput.value === ''){
+    errorMessageEmail.setAttribute('class', 'display-span')
+    errorMessageEmail2.removeAttribute('class');
+    errorMessageEmail2.setAttribute('class', 'display-none'); 
+    emailInput.setAttribute('id', 'border-color')
+  } else if (emailInput.value.match(regex) === null){
+    errorMessageEmail.removeAttribute('class');
+    errorMessageEmail.setAttribute('class', 'display-none');
+    errorMessageEmail2.setAttribute('class', 'display-span')
+    emailInput.setAttribute('id', 'border-color')
+  } else {
+    errorMessageEmail.removeAttribute('class');
+    errorMessageEmail2.removeAttribute('class');
+    errorMessageEmail.setAttribute('class', 'display-none');
+    errorMessageEmail2.setAttribute('class', 'display-none');
+    emailInput.removeAttribute('id'); 
+  }
+};
+
+const errorMessagePassword = document.querySelector('.enter-password');
+const passwordInput = document.querySelector('.password-input');
+passwordInput.onblur = function (e) {
+  if (passwordInput.value === ''){
+    errorMessagePassword.setAttribute('class', 'display-span')
+    passwordInput.setAttribute('id', 'border-color')
+  } else {
+    errorMessagePassword.removeAttribute('class');
+    errorMessagePassword.setAttribute('class', 'display-none');
+    passwordInput.removeAttribute('id'); 
+  }
+}
+
+const eyeOff = document.querySelector('.eye-off')
+const eyeOn = document.querySelector('.eye-on')
+
+eyeOff.onclick = function (e) {
```

Comment:
이벤트 객체를 함수 내부에서 사용하지 않는 경우에는 써주지 않는게 좋습니다!

```suggestion
eyeOff.onclick = function () {
```